include/nudb/impl/basic_store.ipp

```diff
@@ -217,7 +216,7 @@ insert(
     if(ecb_)
     {
         ec = ec_;
-        return false;
+        return;
     }
     // Data Record
     if(size == 0)
```

Comment:
With some param values, exceptions are thrown (eg. size outside range) and with some params, eg. key, an error code is set. As a user, I would prefer the consistency of only having to inspect an error code or catch an exception but not both.